GladiusEx.toc

```diff
@@ -1,8 +1,8 @@
-﻿## Interface: 70200
+﻿## Interface: 70300
 ## Title: GladiusEx
 ## Notes: Arena unit frames
 ## SavedVariables: GladiusExDB
-## OptionalDeps: LibStub, LibFunctional-1.0, Ace3, DRData-1.0, LibSharedMedia-3.0, AceGUI-3.0-SharedMediaWidgets, AceGUI-3.0-Spell-EditBox, LibGroupInSpecT-1.1, LibCooldownTracker-1.0, LibDualSpec-1.0, LibRangeCheck-2.0, LibDispellable-1.0, Clique, Masque
+## OptionalDeps: LibStub, LibFunctional-1.0, Ace3, DRData-1.0, LibSharedMedia-3.0, AceGUI-3.0-SharedMediaWidgets, AceGUI-3.0-Spell-EditBox, LibGroupInSpecT-1.1, LibCooldownTracker-1.0, LibDualSpec-1.0, LibRangeCheck-2.0, LibDispellable-1.0, Clique, Masque, AceGUI-3.0-SharedMediaWidgets
 

```

Comment:
AceGUI-3.0-SharedMediaWidgets was already on the list, it is duplicated now

Reply:
Interesting. Will remove.